queries.yaml

```diff
@@ -104,7 +118,6 @@ layers:
     area-inclusion-threshold: 1
   buildings:
     template: buildings.jinja2
-    start_zoom: 13
```

Comment:
Where is this handled going forward, or is it only on a per-feature `min_zoom` basis now?